Sources/StreamChat/Utils/Codable+Extensions.swift

```diff
@@ -167,6 +167,24 @@ extension DateFormatter {
     }
 }
 
+extension ISO8601DateFormatter {
+    func dateWithMicroseconds(from string: String) -> Date? {
```

Comment:
hmm, could this logic affect the parsing time?

Reply:
I did profiling:
1. Demo app in logged out state
2. Log in with chewbacca
3. Open release 4.63.0 channel, scroll to the very first message
4. Check how much time the dateWithMicroseconds takes, compare it with original 'date(from:)'

In both cases the result was 48 ms which means the additional parsing is negligible compared to what the formatter takes.